tutorial/chapter1.md

```diff
@@ -0,0 +1,200 @@
+<p align="right">
+  <img width="25%" height="25%"src="../images/klepsydra_logo.jpg">
+</p>
+
+# Chapter 1
+
+## Technical requirements
+
+Installation of Klepsydra Core, Klepsydra Robotics, Optionally ROS, DDS.
+
+Klepsydra is installed (by default) to /usr/local by CMake. To develop an application using Klepsydra use the cmake command `find_package(Klepsydra)` in the CMakeLists file. With this command, the Klepsydra includes are available in the variable `KLEPSYDRA_INCLUDE_DIRS` and the libraries are available using the variable `KLEPSYDRA_CORE_LIBRARIES`. The CMakeLists file provided for compiling the examples in the tutorial demonstrate how these two variables are used.
+
```

Comment:
Klepsydra core now uses modern CMake and linking to the core libraries should be sufficient. Using the Klepsydra_include_dirs may not be necessary.

Reply:
Done